### PR TITLE
PIM-6781: improve system info.txt file

### PR DESCRIPTION
**Pull request by aRn0D**
**Description (for Contributor and Core Developer)**

- [x]  Put the "PHP extensions" and "Registered extensions" at the end.
- [x] Display the number of scopable/localizable/scopable and localizable attributes
 
**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | yes
| Added Behats                      | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

File: src/Pim/Bundle/AnalyticsBundle/Resources/config/queries.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    pim_analytics.query.count_scopable_attribute:
+        class: '%pim_analytics.query.count_scopable_attribute.class%'
+        arguments:
+            - '@doctrine.orm.entity_manager'
+
+    pim_analytics.query.count_localizable_scopable_attribute:
+        class: '%pim_analytics.query.count_localizable_scopable_attribute.class%'
+        arguments:
+            - '@doctrine.orm.entity_manager'
```
Comment (solivier) [inline src/Pim/Bundle/AnalyticsBundle/Resources/config/queries.yml]: missing new line

File: src/Pim/Bundle/AnalyticsBundle/Doctrine/Query/CountScopableAttribute.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<?php
+declare(strict_types=1);
+
+namespace Pim\Bundle\AnalyticsBundle\Doctrine\Query;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Pim\Component\Catalog\Model\AttributeInterface;
+
+/**
+ *
```
Comment (solivier) [inline src/Pim/Bundle/AnalyticsBundle/Doctrine/Query/CountScopableAttribute.php]: remove line

File: src/Pim/Bundle/AnalyticsBundle/Doctrine/Query/CountScopableAndLocalizableAttribute.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<?php
+declare(strict_types=1);
+
+namespace Pim\Bundle\AnalyticsBundle\Doctrine\Query;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Pim\Component\Catalog\Model\AttributeInterface;
+
+/**
+ *
```
Comment (solivier) [inline src/Pim/Bundle/AnalyticsBundle/Doctrine/Query/CountScopableAndLocalizableAttribute.php]: remove line

**Comment by jmleroux** [comment]
Would it be possible to have also the version (tag) of the different dependencies?

**Comment by aRn0D** [comment]
@jmleroux this was not asked for this card :) You should create another card for that.

**Pull request merged**